packages/server/server.ts

```diff
@@ -1,5 +1,4 @@
-// Datadog APM, must be first import (disabled for now)
-// import './tracer'
+import 'dd-trace/init'
```

Comment:
side effects on imports are the root of all evil :wink:. 

plus, we probably don't wanna do much tracing in development.

if we initialize it here in our own code, it should be more understandable to our future selves
```ts
import tracer from 'dd-trace'
if (PROD) {
  tracer.init()
}
```

Reply:
I agreed that we don't need tracing on local so the PROD condition makes sense. Curious about the import stuff though, what's your take on the [suggestion from DD documentation](https://docs.datadoghq.com/tracing/setup_overview/setup/nodejs/?tab=containers#adding-the-tracer-in-code) where they suggest to put initialization in a different file?

Reply:
@mattkrick